mailpoet/assets/js/src/sending-paused-notices-resume-button.ts

```diff
@@ -55,6 +55,7 @@ const resumeSendingIfAuthorized = (fromAddress: string | null) => isValidFromAdd
 // jQuery-based notice implementation which doesn't trigger pure-JS added listeners
 jQuery(($) => {
   $(document).on('click', '.notice .mailpoet-js-button-resume-sending', (e) : void => {
-    await resumeSendingIfAuthorized(e.target.value as string);
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    resumeSendingIfAuthorized(e.target.value as string);
```

Comment:
And here adding `void` should help to avoid the eslint-disable, I guess.

Reply:
Addressed in [364b994](https://github.com/mailpoet/mailpoet/pull/3968/commits/364b994295e38917a7e0a01bdb855664dab0742a)